truffle/src/main/ruby/core/exception.rb

```diff
@@ -189,12 +189,11 @@ def initialize(*args)
 end
 
 class NoMethodError < NameError
-  attr_reader :args
 
   def initialize(*arguments)
     super(arguments.shift)
     @name = arguments.shift
```

Comment:
@eregon I don't think this `initialize` method works as is. I will update this method.